docs/docs/tutorial/part-1/index.mdx

```diff
@@ -88,6 +88,17 @@ gatsby new
 
 <Announcement style={{marginBottom: "1.5rem"}}>
 
+You can also generate a Gatsby site without needing to go through all the prompts. In this case, the the defaults will be used except for the site name which you'll need to pass.
```

Comment:
We should remove this whole part as it's not relevant for the tutorial. We want people to go through all the steps and not bail out early here